sns_ik_lib/include/sns_ik/fosns_velocity_ik.hpp

```diff
@@ -36,12 +36,12 @@ namespace sns_ik {
 
 class FOSNSVelocityIK : public FSNSVelocityIK {
   public:
-    FOSNSVelocityIK(int dof, Scalar loop_period);
+    FOSNSVelocityIK(int dof, double loop_period);
     virtual ~FOSNSVelocityIK() {};
 
     // Optimal SNS Velocity IK
-    virtual Scalar getJointVelocity(VectorD *jointVelocity, const std::vector<Task> &sot,
-                  const VectorD &jointConfiguration);
+    virtual double getJointVelocity(Eigen::VectorXd *jointVelocity, const std::vector<Task> &sot,
+                  const Eigen::VectorXd &jointConfiguration);
```

Comment:
It's fine to explicitly state the namespace of the Eigen Vector, but then remove the `using namespace` declaration above.